cicecore/drivers/nuopc/cmeps/CICE_RunMod.F90

```diff
@@ -201,6 +201,17 @@ subroutine ice_step
          call ice_timer_start(timer_column)  ! column physics
          call ice_timer_start(timer_thermo)  ! thermodynamics
 
+         !-----------------------------------------------------------------
+         ! snow redistribution and metamorphosis
+         !-----------------------------------------------------------------
+
+         if (tr_snow) then         ! advanced snow physics
+            do iblk = 1, nblocks
+               call step_snow (dt, iblk)
+            enddo
+            call update_state (dt) ! clean up
+         endif
+
```

Comment:
This block also is present at LNS 324:329. When should it actually be called? I was following the standalone and dmi placement.

Reply:
Good catch. The merge seemed to cause this. I will look to see where these should be or if it matters.